protocol/protocol.proto

```diff
@@ -190,6 +191,15 @@ message MemberPayload {
         bytes token_id = 3;
     }
 
+    message Pin { 
```

Comment:
backwards compatibility: If a new client (post-this-PR) sends this payload to an old stream node on omega, is that going to work?

Reply:
The request will fail until the node is upgraded.